spec/models/account_spec.rb

```diff
@@ -1,34 +1,87 @@
 describe Account do
-  before(:each) do
-    @user = FactoryGirl.create(:account_user)
-    @group = FactoryGirl.create(:account_group)
+  let(:user) { FactoryGirl.create(:account_user) }
+  let(:group) { FactoryGirl.create(:account_group) }
+
+  context 'when host has users and groups' do
+    let(:host) { FactoryGirl.create(:host) }
+    let(:test_xml) do
+      REXML::Document.new("<miq>
+                          <accounts>
+                          <users><user name='test_user'/></users>
+                          <groups><group name='test_group'/></groups>
+                          </accounts>
+                          </miq>")
+    end
+
+    it '.add_elements' do
+      user.class.add_elements(host, test_xml)
+      expect(host.users.find_by(:name => 'test_user')).to be
+      expect(host.groups.find_by(:name => 'test_group')).to be
```

Comment:
Prefer not to use `be` (with no predicate). Please use something that indicates presence/existence (ie `be_present`, `expect(host.users.exists?(:name => 'test_user')).to be_truthy`...)